public/app/features/alerting/unified/components/rule-viewer/AlertRuleMenu.tsx

```diff
@@ -86,7 +86,7 @@ const AlertRuleMenu = ({
 
   const menuItems = (
     <>
-      {canPause && rulerRule && groupIdentifier.groupOrigin === 'grafana' && (
+      {canPause && isGrafanaRulerRule(rulerRule) && groupIdentifier.groupOrigin === 'grafana' && (
         <MenuItemPauseRule rule={rulerRule} groupIdentifier={groupIdentifier} onPauseChange={onPauseChange} />
       )}
```

Comment:
@tomratcliffe We render the `MenuItemPauseRule` only when we know the rule is Grafana ruler rule, so we could narrow down the type definition